lighthouse-cli/index.js

```diff
@@ -77,11 +98,11 @@ if (cli.flags.listAllAudits) {
   process.exit(0);
 }
 
-const url = cli.input[0] || 'https://m.aliexpress.com/';
-const outputMode = cli.flags.output || Printer.OUTPUT_MODE.pretty;
-const outputPath = cli.flags.outputPath || 'stdout';
-const flags = cli.flags;
-const config = (cli.flags.configPath && require(cli.flags.configPath)) || null;
+const url = cli._[0];
```

Comment:
mostly laziness, I think. It's nice for development and for showcasing without having a specific site in mind to test, and easy to catch if you need to run lighthouse against a specific url